### PR TITLE
Bump document version to reflect per-document library changes

### PR DESCRIPTION
**Pull request by elfenlaid**

### CODE REVIEW

File: .changeset/blue-houses-punch.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@sketch-hq/sketch-file-format': minor
+'@sketch-hq/sketch-file-format-ts': minor
```
Comment (elfenlaid) [inline .changeset/blue-houses-punch.md:3]: Actually I'm not sure we need to bump in to `minor`, as https://github.com/sketch-hq/sketch-document/pull/213/files already bumped the file format versions 🤔

Comment (jedrichards) [reply .changeset/blue-houses-punch.md:3]: Yeap I think this PR not needed, you already have a changeset in the other PR. You can just merge the [release PR](https://github.com/sketch-hq/sketch-document/pull/214) and good to go I think.

Comment (elfenlaid) [reply .changeset/blue-houses-punch.md:3]: @jedrichards Hmm, but we still need [meta.schema.yaml](https://github.com/sketch-hq/sketch-document/pull/215/files#diff-ad31e01dc0226a07e7bf33ca2bef7df78e4932fedf15d2c0bc8cc9ee92311441) changes, don't we?

Comment (jedrichards) [reply .changeset/blue-houses-punch.md:3]: Oh I see there is a little change in this PR, sorry was confused. Thought this PR was just the changeset addition.

In which case its fine to keep it and the extra bumps. Multiple minor bumps in one release will just result in one +1 to the actual minor version.

Comment (lrossi) [reply .changeset/blue-houses-punch.md:3]: Correct, I asked Egor to add a new document version cause we're bumping it on the Mac side.

**Review by lrossi — APPROVED** [review]

File: .changeset/blue-houses-punch.md (hunk 1 above)
Comment (elfenlaid) [reply .changeset/blue-houses-punch.md:3]: Awesome, tho someone needs to merge the PR as I don't have merge rights 🙈

**Pull request merged**